tests/unittests/config/test_cc_ansible.py

```diff
@@ -409,7 +413,7 @@ def test_ansible_pull(self, m_subp1, m_subp2, m_which, cfg, expected):
     @mock.patch(M_PATH + "validate_config")
     def test_do_not_run(self, m_validate):
         """verify that if ansible key not included, don't do anything"""
-        cc_ansible.handle("", {}, get_cloud(), None)  # pyright: ignore
+        cc_ansible.handle("", {}, get_cloud(), [])  # pyright: ignore
```

Comment:
```suggestion
        cc_ansible.handle("", {}, get_cloud(), [])
```
Typing ignore no longer needed